custom_components/hacs/tasks/activate_categories.py

```diff
@@ -18,7 +18,7 @@ class Task(HacsTask):
 
     stages = [HacsStage.SETUP]
 
-    def execute(self) -> None:
+    async def async_execute(self) -> None:
```

Comment:
```suggestion
    def execute(self) -> None:
```
This task only reset and add elements to a set, that is safe?

Reply:
It is likely safe, but is less risky to do in the main thread if we can so we don't have to think about if the GIL is making the structure safe or not.

Reply:
Fair enough :+1: